### PR TITLE
docs: Update Acceptance policy

### PR DESCRIPTION
**Pull request by yiannistri**
**What changed?**
Updated Acceptance policy.

**Why was this change made?**
To make PR requirements more explicit.

**How was this change implemented?**
N/A

**How did you validate the change?**
N/A

**Release notes**
N/A

**Documentation Changes**
N/A

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -23,6 +23,11 @@ These things will make a PR more likely to be accepted:
 - tests for old code!
 - new code and tests follow the conventions in old code and tests
 - a good commit message (see below)
+- all code must abide [Go Code Review Comments](https://github.com/golang/go/wiki/CodeReviewComments)
```
Comment (LappleApple) [inline CONTRIBUTING.md:26]:
```suggestion
- all code must abide by [Go Code Review Comments](https://github.com/golang/go/wiki/CodeReviewComments)
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -23,6 +23,11 @@ These things will make a PR more likely to be accepted:
...
+- names should abide [What's in a name](https://talks.golang.org/2014/names.slide#1)
```
Comment (LappleApple) [inline CONTRIBUTING.md:27]:
```suggestion
- names should abide by [What's in a name](https://talks.golang.org/2014/names.slide#1)
```

**Review by LappleApple — COMMENTED** [review]
two small nits

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -23,6 +23,11 @@ These things will make a PR more likely to be accepted:
 - tests for old code!
 - new code and tests follow the conventions in old code and tests
 - a good commit message (see below)
+- all code must abide by [Go Code Review Comments](https://github.com/golang/go/wiki/CodeReviewComments)
+- names should abide by [What's in a name](https://talks.golang.org/2014/names.slide#1)
+- code must build on both Linux and Darwin, via plain `go build`
```
Comment (yitsushi) [inline CONTRIBUTING.md:28]: That's a tricky one to enforce. It makes sense, but as a random person if I try to join to contribute that may make some uncertainties as I have no mac where I can test and build.

**Review by yitsushi — APPROVED** [review]

File: CONTRIBUTING.md (hunk 3 above)
Comment (yiannistri) [reply CONTRIBUTING.md:28]: Agreed, this is not meant to be a strict/hard requirement but rather an acknowledgment that a simple `go build` should be sufficient.

**Pull request merged**